packages/react-examples/.storybook/preview.js

```diff
@@ -125,7 +121,17 @@ function generateStoriesFromExamples(key, stories, req) {
     return;
   }
 
-  const componentName = segments.length === 3 ? segments[1] : `${segments[2]} (${segments[1]})`;
+  /** @type {string} */
+  let componentName;
+  if (segments.length === 3) {
```

Comment:
**q:** what does 3 mean? can you abstract this to self describing constant please?

Reply:
The comment below shows which case this refers to. I'm not even sure what a constant could possibly be called in this case...